Products/ZenHub/zenhubworker.py

```diff
@@ -299,6 +300,12 @@ def buildOptions(self):
             '--hubpassword', dest='hubpassword', default='zenoss',
             help="password to use when connecting to ZenHub",
         )
+        self.parser.add_option(
+            '--hub-connect-timeout', dest='hub_connect_timeout',
```

Comment:
can we make this hub-response-timeout so the "connection established" message isn't as confusing?